### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ole-na**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
 # APP PhotoNavTex
-App for nautical warning messages
+
+At the end of neufische intensive coding bootcamp (https://www.neuefische.de/en/weiterbildung/java/hamburg) I have implemented this app for nautical warning messages.
+
+NAVTEX is  an international system for maritime safety information (MSI) which includes navigational warnings, weather forecasts and weather warnings, search and rescue notices and similar information to ships. PhotoNavTex helps to indicate two types of these warnings: **A** (navigational warnings) and **D** (Search And Rescue information and pirate attack warnings). Therefore it displays the position of saved warnings on the OpenStreetMap as basis map and on OpenSeaMap as overlay.
```
Comment (Cathesso) [inline README.md:5]: I would say 
"[...] similar information for ships."
and "PhotoNavTex helps to indicate two types of warnings [...]"
and "It displays the position of saved warnings on the [...]"

File: README.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 
 ## NAVTEX 
 
-NAVTEX is an international, automated system for instantly distributing maritime safety information (MSI) which includes navigational warnings, weather forecasts and weather warnings, search and rescue notices and similar information to ships.
+### NAVTEX messages 
 
 The messages are coded with a header code identified by the using single letters of the alphabet to represent broadcasting stations, type of messages, and followed by two figures indicating the serial number of the message.
```
Comment (Cathesso) [inline README.md:11]: I would say: "The messages are coded with a letter representing the broadcasting station, the type of message and then followerd by two signs which indicate the messages serial number."

**Review by Cathesso — COMMENTED** [review]
Looks great! 🚀

File: README.md (hunk 2 above)
Comment (ole-na) [reply README.md:11]: ok

**Pull request merged**